File: cpp/Column name from a given column number.cpp
class Solution{
    public:
    string colName (long long int n)
    {
        // your code here
        string s = "";
        while(n-->0){
            s+='A'+n%26;
            n=n/26;
        }
        reverse(s.begin(),s.end());
        return s;
    }
};

File: java/Column name from a given column number.java
class Solution
{
    String colName (long n)
    {
        // your code here
        StringBuilder sb = new StringBuilder();
        
        while(n-->0){
            sb.append((char)('A'+n%26));
            n = n/26;
        }
        
        return sb.reverse().toString();
    }
}